src/Mono.Android/PublicAPI/API-34/PublicAPI.Unshipped.txt

```diff
@@ -6918,7 +6918,7 @@ Android.Appwidget.AppWidgetProviderInfo.WidgetCategory.set -> void
 Android.Appwidget.AppWidgetProviderInfo.WidgetFeatures.get -> Android.Appwidget.WidgetFeatures
 Android.Appwidget.AppWidgetProviderInfo.WidgetFeatures.set -> void
 Android.Appwidget.ResizeMode
-Android.Appwidget.ResizeMode.Both = 3 -> Android.Appwidget.ResizeMode
+Android.Appwidget.ResizeMode.Both = Android.Appwidget.ResizeMode.Horizontal | Android.Appwidget.ResizeMode.Vertical -> Android.Appwidget.ResizeMode
```

Comment:
This makes for an "interesting" change: using `[Flags]` means that `PublicAPI.Unshipped.txt`. has "more readable" output, using named constants instead of underlying integers.  Nice!

Alas, we can't use "presence of a raw value in `PublicAPI.Unshipped.txt` as an "alternate verification", because "used of named constants" only happens for `|`ed values.